pkg/bootstrapper/bootstrapper.go

```diff
@@ -323,7 +328,11 @@ func (wmcb *winNodeBootstrapper) parseIgnitionFileContents(ignitionFileContents
 	// and write it to the destination path
 	for _, ignFile := range configuration.Storage.Files {
 		if filePair, ok := filesToTranslate[ignFile.Node.Path]; ok {
-			newContents, err := wmcb.translateFile(ignFile.Contents.Source, filePair.translationFunc)
+			if ignFile.Contents.Source == nil {
```

Comment:
Should this be done directly below the `for _, ignFile := range configuration.Storage.Files {` line?
No point in doing `if filePair, ok := filesToTranslate[ignFile.Node.Path]; ok {` if the file is empty

Reply:
Having empty file contents is not invalid per se and some files like that could be included in the Ignition config we get from the MCS.
My reasoning here was that we really just care about the files in the `filesToTranslate` map, which must not be empty, and we can fail in that case.